application/views/view_log/partial/log_ajax.php

```diff
@@ -32,8 +32,10 @@ function echo_table_col($row, $name) {
 		case 'WWFF':    echo '<td>' . ($row->COL_WWFF_REF ?? '') . '</td>'; break;
 		case 'POTA':    echo '<td>' . ($row->COL_POTA_REF ?? '') . '</td>'; break;
 		case 'Grid':
-            $ci->load->library('qra');
-            echo '<td>' . ($ci->qra->echoQrbCalcLink($row->station_gridsquare, $row->COL_VUCC_GRIDS, $row->COL_GRIDSQUARE)) . '</td>'; break;
+				if(!$this->load->is_loaded('Qra') {
+					$this->load->library('Qra');
```

Comment:
```suggestion
				if(!$ci->load->is_loaded('Qra') {
					$ci->load->library('Qra');
```

Reply:
And even a closing parenthesis missing :)